chapters/packages.tex

```diff
@@ -326,3 +326,122 @@ \section{External resources}\label{external-resources}
 \filename{modelica://Modelica/Mechanics/C.jpg} is illegal.
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
+
+\section{Multilingual Descriptions}\label{multilingual-descriptions}
+
+\begin{nonnormative}
+Descriptive texts in a model or library are usually formulated in English.
+To support users who are not so familiar with this, a Modelica tool can support the translation of these texts into any language.
+This should be done using external files, so that no modification of the actual Modelica text is required.
+\end{nonnormative}
+
+The files to support translation must be provided by the developer of the library.
+They must be stored in the resources directory \filename{modelica://<LibraryName>/Resources/Language/}.
+
+Two kind of files have to be provided:
+\begin{enumerate}
+\item Template file \filename{<LibraryName>.pot}, as the resource \filename{modelica://<LibraryName>/Resources/Language/<LibraryName>.pot}.
+It contains all necessary information to translate all descriptions, but no translations.
+The pattern \filename{<LibraryName>} denotes the toplevel class name of the library.
+\item One file for each supported language with the name \filename{<LibraryName>.<language>.po}, as the resource \filename{modelica://<LibraryName>/Resources/Language/<LibraryName>.<language>.po}.
+This file is a copy of the associated template file, but extended with the translations in the specified language.
+The pattern \filename{<language>} stands for the ISO 639-1 language code, e.g. \filename{de} or \filename{en}.
```

Comment:
Missing guard for period not at end of sentence:
```suggestion
The pattern \filename{<language>} stands for the ISO 639-1 language code, e.g., \filename{de} or \filename{en}.
```

Reply:
change was comtained in another commit.

Reply:
Yes.  Resolving.